_pytest/helpconfig.py

```diff
@@ -20,6 +20,12 @@ def pytest_addoption(parser):
     group.addoption('--debug',
                action="store_true", dest="debug", default=False,
                help="store internal tracing debug information in 'pytestdebug.log'.")
+    # support for "--overwrite-ini ININAME=INIVALUE" to overrides ini values from the command line
+    # so that one can do e.g. "-o xfail_strict=True". This is useful for specifying options for which
+    # there is no separate command-line flag, you can find ini-options by py.test --help
+    group._addoption('-o', '--override-ini',
+                    nargs='*', dest="override_ini",
+                    help="overrides ini values which do not have a separate command-line flag")
```

Comment:
i suggest: overrides values from the ini file.  Example '-o xfail_strict=True'. 

because it even works if there are other command line flags ;)